app/src/main/java/chat/rocket/android/chatrooms/presentation/ChatRoomsPresenter.kt

```diff
@@ -238,6 +244,14 @@ class ChatRoomsPresenter @Inject constructor(
         }
     }
 
+    private fun chatRoomTimestamp(chatRoom: ChatRoom): Long? {
+        return if (settings.hasShowLastMessage() && settings.showLastMessage()) {
+            chatRoom.lastMessage?.timestamp ?: chatRoom.updatedAt
```

Comment:
@rafaelks is this OK? if a room doesn't have last message recorded yet, we will use the `updatedAt` for this room

Reply:
@luciofm Yes, perfect! 👌